app/models/requests/MovementRequest.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.requests
+
+import forms._
+import play.api.libs.json.{Json, OFormat}
+
+case class MovementRequest(
+  choice: String,
```

Comment:
Why do you use String here ?

Reply:
Would be nicer as a `MovementType` enum

Reply:
Would force that it can only be ARRIVE or DEPART

Reply:
Done